app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -94,73 +101,16 @@ If you don’t want to come to England to study you can apply to train for inter
 
 For teachers who qualified outside the UK, as well as those with teaching experience, [other routes to qualified teacher status (QTS)](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk) are also open.
 
-## Need to know
-
-When you plan your training, bear in mind that:
-
-- international candidates can expect to pay between £9,250 and £32,000, plus
-living costs, to train (depending on your immigration status and the subject you train to teach, you may get some financial support)
-- training lasts one academic year full-time, or longer part-time
-- you can apply for paid teaching jobs as soon as you successfully complete your training
-- you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher
-- you can update, extend or switch your visa to live and work in the UK long-term
-
-
-
-## Apply for teacher training
-
-We can help you make an application for teacher training in England. Sign up for a teacher training adviser to get help right away and throughout the application process.
+## Check your eligibility and get help
 
-### Get support and advice
+Before you apply, you must [check your eligibility](/non-uk-teachers/non-uk-qualifications). If you gained your qualifications outside the UK, we can help you check they meet the required standard. 
 
-For help applying, you can:
+For help with qualifications and completing your application for teacher training, you can:
 
 * sign up to [get a teacher training adviser](/teacher-training-adviser/sign_up/identity) – you’ll be asked for your phone number, email and location so a dedicated adviser can call you back in your time zone
```

Comment:
Do we want to encourage them to use the live chat as an alternative? Because I think that might be easier/less off-putting than making an international call?